includes/Preferences.php

```diff
@@ -85,10 +85,14 @@ static function getPreferences( $user, IContextSource $context ) {
 			}
 		}
 
+		## Make sure that form fields have their parent set. See SUS-1853
+		$dummyForm = new HTMLForm( array(), $context );
+
 		## Prod in defaults from the user
 		foreach ( $defaultPreferences as $name => &$info ) {
 			$prefFromUser = self::getUserProperty( $name, $info, $user );
 			$field = HTMLForm::loadInputFromParameters( $name, $info ); // For validation
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)